### PR TITLE
chore: update integerationapp for newer OS env

### PR DESCRIPTION
**Pull request by KazuCocoa**
IntegerationApp has a few deprecations. This PR fixed them.

I have tested these changes with simulators and a real device.

The notification thing is to show below once.

Screenshot 2024-04-19 at 12 38 01 AM

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
is it `chore` or `fix`?

File: WebDriverAgentTests/IntegrationApp/Info.plist [hunk 1]
```diff
@@ -22,12 +22,12 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>Yo Yo</string>
```
Comment (mykola-mokhnach) [inline WebDriverAgentTests/IntegrationApp/Info.plist:26]: Yo!

**Review by mykola-mokhnach — APPROVED** [review]
thanks for updating that @KazuCocoa

**Comment by KazuCocoa** [comment]
This is for integration app's kind of "update" for newer OSs, so not "fix" which actual WDA behavior. Hm, let me modify the title as update integration app

**Pull request merged**